app/Web/Mail.php

```diff
@@ -71,7 +71,8 @@ public function to(string $mail)
      */
     public function subject(string $text)
     {
-        $this->subject = htmlentities($text);
+        $this->subject = utf8_decode($text);
```

Comment:
This line is actually useless, the `$this->subject` value is overwritten the next line